tests/conftest.py

```diff
@@ -376,27 +350,9 @@
     os.path.join(CUR_PATH, "data", "aea-config.example.yaml"),
     os.path.join(CUR_PATH, "data", "aea-config.example_w_keys.yaml"),
     os.path.join(CUR_PATH, "data", "aea-config.example_multipage.yaml"),
-    os.path.join(FETCHAI_PREF, "agents", "aries_alice", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "aries_faber", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "car_data_buyer", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "car_detector", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "confirmation_aea_aw1", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "erc1155_client", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "erc1155_deployer", AGENT_YAML),
     os.path.join(FETCHAI_PREF, "agents", "generic_buyer", AGENT_YAML),
     os.path.join(FETCHAI_PREF, "agents", "generic_seller", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "gym_aea", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "ml_data_provider", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "ml_model_trainer", AGENT_YAML),
     os.path.join(FETCHAI_PREF, "agents", "my_first_aea", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "registration_aea_aw1", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "simple_service_registration", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "tac_controller", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "tac_controller_contract", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "tac_participant", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "tac_participant_contract", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "thermometer_aea", AGENT_YAML),
-    os.path.join(FETCHAI_PREF, "agents", "thermometer_client", AGENT_YAML),
```

Comment:
Why remove these constants?

Reply:
aea tests to refer limited set of packages
to allow move some packages out